handler.go

```diff
@@ -118,6 +118,8 @@ func NewRouter(handler *Handler) *mux.Router {
 	router.HandleFunc("/index/{index}/frame/{frame}", handler.handlePostFrame).Methods("POST")
 	router.HandleFunc("/index/{index}/frame/{frame}", handler.handleDeleteFrame).Methods("DELETE")
 	router.HandleFunc("/index/{index}/frame/{frame}/attr/diff", handler.handlePostFrameAttrDiff).Methods("POST")
+	router.HandleFunc("/index/{index}/frame/{frame}/attr", handler.handleGetFrameAttr).Methods("GET")
```

Comment:
all of these handler routes are kept in alphabetical order by endpoint. the four you added need to be rearranged.